plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestShowStats.java

```diff
@@ -337,6 +337,20 @@ public void testShowStatsForColumnExpression()
                         "   (null, null, null, null, 25, null, null)");
     }
 
+    @Test
+    public void testShowStatsWithWith()
```

Comment:
Could you also add examples like `SHOW STATS FOR (TABLE region)`, `SHOW STATS FOR (VALUES 1, 2, 3)`, `SHOW STATS FOR (TABLE t1 INTERSECT TABLE t2)`, which weren't supported before? Generally, with query body other than `QuerySpecification`.

Reply:
added